src/Kernel.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
Why its here ? we commit config implementation, without config files in a task about notification sending? :)